packages/graphql-language-server/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "graphql-language-server",
+  "version": "2.2.0",
+  "main": "./dist/cli.js",
+  "browser": "./dist/worker.js",
+  "types": "./dist/types/GraphqlLanguageServer.ts",
+  "bin": "./dist/cli.js",
+  "license": "Apache-2.0",
+  "keywords": [
+    "graphql",
+    "stardog",
+    "language server",
+    "rdf",
+    "query",
+    "knowledge graph"
+  ],
+  "author": {
+    "name": "Stardog Union",
+    "url": "https://stardog.com"
+  },
+  "contributors": [
+    {
+      "name": "Jason Rogers",
+      "email": "jason@stardog.com"
+    },
+    {
+      "name": "Ty Soehngen",
+      "email": "ty@stardog.com"
```

Comment:
> I'm having trouble using Studio with this. Is it because of the conflicting package name? Even when I change that to `stardog-graphql-language-server` in this package's package.json and the dep I add to package.json in your Studio branch it fails (after building, linking, etc) at `yarn` in Studio. Does it work for you?

Yeah, it works for me. You need to do a bunch of linking to make it work. First, you have to run `yarn` inside of the `stardog-language-servers`repo. Next, you have to go into the `stardog-language-utils` directory and run `yarn build` followed by `yarn link`. Then you have to go into the `graphql-language-server` directory and run `yarn link stardog-language-utils` followed by `yarn build` and `yarn link`. At that point, you will have to run `yarn link stardog-language-utils` and `yarn link graphql-language-server` inside of the Studio repo, and it should work.

Reply:
I don't know what new emails (if any) they'd like us to use (though I've messaged them to ask), and I don't want to remove them as contributors, so I'm gonna leave this for now, if that's okay.

Reply:
Yep, just curious